Source/SIMPLib/Common/SIMPLArray.hpp

```diff
@@ -172,6 +184,22 @@ class SIMPLArray
     return m_Array == rhs.m_Array;
   }
 
+  /**
+   * @brief Converts to another container type. The output type that is being used needs to have the "push_back()" method implemented.
+   *
+   *   For STL containers this includes Vector, List, Deque. QVector and QList will also work.
+   */
+  template <typename OutContainerType>
+  OutContainerType toContainer()
+  {
+    OutContainerType dest;
```

Comment:
If you don't mind losing QList (which is suggested to avoid like the plague), I'm going to say the same thing I mentioned on another pull request.  Pass in the dimentions to `OutContainerType dest(m_Array.size());` and keep your [] operators for assignment and copying.  Every other container mentioned in the description allows that pattern, and the most likely conversions (std::vector and QVector) do so optimally.  std::list and std::deque are slower when it comes to random access but are similarly unlikely conversions from an array.

Reply:
All requests implemented.